packages/elements/src/components/ino-tooltip/ino-tooltip.tsx

```diff
@@ -80,10 +80,6 @@ export class Tooltip implements ComponentInterface {
     await this.create();
   }
 
-  async disconnectedCallback() {
-    await this.dispose();
```

Comment:
Why did you remove this call?

Reply:
This call would destroy the tooltip right after the first hover/click (when it is removed from the DOM), so the tooltip would only be visible once. The tooltip should only be destroyed when the target element is destroyed (this is not covered in this MR). Same for the popover.